Soundify/Soundify/Scenes/Home/HomeViewController.swift

```diff
@@ -9,4 +9,59 @@
 import UIKit
 
 final class HomeViewController: UIViewController {
+    
+    private var albums: [Album]?
```

Comment:
```suggestion
    private var albums: [Album] = []
```
array mặc định để giá trị empty nhé